programs/pkey/rsa_sign.c

```diff
@@ -57,12 +57,23 @@ int main( int argc, char *argv[] )
     FILE *f;
     int ret;
     size_t i;
+#if defined(MBEDTLS_PLATFORM_C)
+    mbedtls_platform_context platform_ctx;
+#endif
     mbedtls_rsa_context rsa;
     unsigned char hash[32];
     unsigned char buf[MBEDTLS_MPI_MAX_SIZE];
     char filename[512];
     mbedtls_mpi N, P, Q, D, E, DP, DQ, QP;
 
+#if defined(MBEDTLS_PLATFORM_C)
+    if( ( ret = mbedtls_platform_setup( &platform_ctx ) ) != 0 )
+    {
+        mbedtls_printf( " failed\n  ! mbedtls_platform_setup returned %d\n\n", -ret );
```

Comment:
We're displaying a negative return value here, yet at line 105 and onwards, are displaying it without the negative. We should be self-consistent within the same program.